lib/tasks/build.js

```diff
@@ -21,9 +21,13 @@ module.exports = function(options) {
 				taskPrefix + 'styles',
 				taskPrefix + 'vendor'
 			],
-			[taskPrefix + 'soyweb', taskPrefix + 'metal']
+			[taskPrefix + 'soyweb']
 		];
 
+		if (!options.skipBundle) {
+			tasks.push(taskPrefix + 'metal');
+		}
+
```

Comment:
Hey @alexandervalencia 

By pushing to `tasks`, the `metal` task will no longer run in parallel with the `soyweb` task. It will end up looking like this.

```
[
	taskPrefix + 'clean',
	[
		taskPrefix + 'front-matter',
		taskPrefix + 'static',
		taskPrefix + 'styles',
		taskPrefix + 'vendor'
	],
	[taskPrefix + 'soyweb'],
	taskPrefix + 'metal'
];
```

But we want it to look like

```
[
	taskPrefix + 'clean',
	[
		taskPrefix + 'front-matter',
		taskPrefix + 'static',
		taskPrefix + 'styles',
		taskPrefix + 'vendor'
	],
	[taskPrefix + 'soyweb', taskPrefix + 'metal'],
];
```

Reply:
@Robert-Frampton got it, will fix. 👍